about.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="https://use.typekit.net/vyw3zyb.css">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <ul class="header-links">
+                <li><img src="assets/images/facebook-icon.svg" alt=""></li>
+                <li><img src="assets/images/twitterIcon.png" alt=""></li>
+                <li>English</li>
+                <li>my page</li>
+                <li>log in</li>
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a></div>
+            <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+            <ul class="nav-links" id="nav-links">
+              <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="icon" alt="close menu icon">
+                <li><a href="index.html">Home</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="">program</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Join</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Sponsor</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
```

Comment:
- [ ] You did a good job making the change as requested of you by the previous reviewer. However, just as in the home page title, kindly align the heading properly so that it matches the provided Behance design.

**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/212383661-308c64e2-54ab-41d2-ad65-396619d8821b.png)


**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/212383876-cba4fe86-553c-4d14-9092-21fa99ecf11e.png)